web/src/components/confirm-form.tsx

```diff
@@ -25,32 +27,52 @@ export default function ConfirmForm(props: Props) {
     isUploadingSentences,
   } = props;
 
+  const countOfSentences = submitted.length;
+  const countOfInvalidated = invalidated.length;
+  const countOfReviewed = validated.length + invalidated.length;
   const readyCount = submitted.length - invalidated.length;
+  const countOfUnreviewed = unreviewed.length;
```

Comment:
For discussion, let me explain why I did it:
* Mainly for code readability
* Better naming in translation files
The translation file has the following (e.g. it will be shown in Pontoon):

`sc-confirm-already-reviewed = -- { $countOfReviewed } sentences are already reviewed. Great job!`

The code has:

`
const countOfReviewed = validated.length + invalidated.length;
        ...
        <Localized id="sc-confirm-already-reviewed" vars={{ countOfReviewed }}>
`

It could be of course as follows:

`
        <Localized id="sc-confirm-already-reviewed" vars={{ countOfReviewed:  validated.length + invalidated.length }}>
`

I might be old-school in these, so it is open for discussion.

Reply:
The last idea in the next topic is most plausible, so I implemented it.

Reply:
Done...

Reply:
Locally tested OK.